turborepo-tests/examples/test_example.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Start by figuring out which example we're testing and its package manager
+example_path=$1
+package_manager=$2
+
+# If you need to do some debugging, you can set these to true
+debug=false
+force=false
+
+# Use the right command for each package manager
+if [ "$package_manager" == "npm" ]; then
+  package_manager_command="npm install"
+  update_turbo_version="npm install turbo@latest -w"
+elif [ "$package_manager" == "pnpm" ]; then
+  package_manager_command="pnpm install"
+  update_turbo_version="pnpm install turbo@latest -w"
+elif [ "$package_manager" == "yarn" ]; then
+  package_manager_command="yarn"
+  update_turbo_version="yarn upgrade turbo@latest -W"
+fi
+
+# All examples implement these two tasks
+# and it's reasonable to assume that they will continue to do so
+turbo_command="turbo build lint --output-logs=errors-only"
+
+# Head into the example directory
+cd $example_path
```

Comment:
Using a monorepo inside another monorepo was problematic before and that's why we added prysk, so that execution would occur in a tmp directory. https://github.com/vercel/turbo/pull/4000

Reply:
Seems to be working well now! Not sure what would have changed from then to now but I'm not running into any problems.